src/utils/usageData.ts

```diff
@@ -140,26 +144,25 @@ export const reset = () => {
  *
  * @returns {void}
  */
-const sendEvent = ({ action, label }: EventAction) => {
+const sendEvent = ({ action, metadata }: EventAction) => {
     const isSendingUsageData = getIsSendingUsageData();
 
     if (isSendingUsageData && insights !== undefined) {
-        logger?.debug(`Sending usage data ${action} ${label}`);
-        insights.trackEvent({
-            name: action,
-            properties: label ? { label } : undefined,
-        });
+        logger?.debug(`Sending usage data ${action}`);
```

Comment:
As it's a debug logger you could also include the full metadata.
```suggestion
        logger?.debug(`Sending usage data ${action}: ${JSON.stringify(metadata)}`);
```